### ISSUE TITLE
presence_of_element_located needs ( Tuple ) for argument

### ISSUE THREAD

**Issue by gaoming714** [SeleniumHQ/seleniumhq.github.io#522]
## 🐛 Bug Report

``` python
Scenario: User can open Create Computer screen              # features/UI_Tests/Create.feature:8
    Given A user has navigated to the BB_Test_Webpage         # steps/Backbase_common_steps.py:6
    And add a new computer is clicked                         # steps/Backbase_common_steps.py:11
    Then the user will be navigated to "Create_Computer" page # steps/web_create_steps.py:5
      Traceback (most recent call last):
......................................
          EC.presence_of_element_located(*element)
      TypeError: __init__() takes 2 positional arguments but 3 were given
```

## To Reproduce

document error.
For this page with Python
https://www.selenium.dev/documentation/en/

## Expected behavior

`first_result = wait.until(presence_of_element_located(By.CSS_SELECTOR, "h3>div"))`
should be
`first_result = wait.until(presence_of_element_located( (By.CSS_SELECTOR, "h3>div") ))`
Because the argument should be an tuple. the current one does not work.

## Environment

* OS: windows.
* Browser: Firefox
* Browser version:

**Comment by diemol** [comment]
@gaoming714 could you please give us more context? I cannot understand what is the bug in the documentation.

**Comment by gaoming714** [comment]
@diemol 
if I run the demo code in the document.
```python
from selenium import webdriver
from selenium.webdriver.common.by import By
from selenium.webdriver.common.keys import Keys
from selenium.webdriver.support.ui import WebDriverWait
from selenium.webdriver.support.expected_conditions import presence_of_element_located

#This example requires Selenium WebDriver 3.13 or newer
with webdriver.Firefox() as driver:
    wait = WebDriverWait(driver, 10)
    driver.get("https://google.com/ncr")
    driver.find_element(By.NAME, "q").send_keys("cheese" + Keys.RETURN)
    first_result = wait.until(presence_of_element_located(By.CSS_SELECTOR, "h3>div"))
    print(first_result.get_attribute("textContent"))
```

I will get an Error

```python
TypeError                                 Traceback (most recent call last)
<ipython-input-7-81c67a2d4c52> in <module>
      4     driver.get("https://google.com/ncr")
      5     driver.find_element(By.NAME, "q").send_keys("cheese" + Keys.RETURN)
----> 6     first_result = wait.until(presence_of_element_located(By.CSS_SELECTOR,"h3>div"))
      7     print(first_result.get_attribute("textContent"))
      8

TypeError: __init__() takes 2 positional arguments but 3 were given

```

the source code for `presence_of_element_located`
```python
class presence_of_element_located(object):
    """ An expectation for checking that an element is present on the DOM
    of a page. This does not necessarily mean that the element is visible.
    locator - used to find the element
    returns the WebElement once it is located
    """
    def __init__(self, locator):
        self.locator = locator

    def __call__(self, driver):
        return _find_element(driver, self.locator)
```

It can be fixed by changing `By.CSS_SELECTOR, "h3>div"` to `(By.CSS_SELECTOR, "h3>div")`

And this is the first demo for Python in the document, I think it deserves more attention.

**Comment by diemol** [comment]
Where is this code? Can you please provide the link?

**Comment by gaoming714** [comment]
link:
https://www.selenium.dev/documentation/en/

the python part

![Image 1](https://user-images.githubusercontent.com/7444968/93095607-e6de0b80-f6d5-11ea-8a26-57a63b83d104.png)

**Comment by diemol** [comment]
Ah ok, now I understand. 
I was able to reproduce it.
I could fix it, but I wanted to ask first if you would like to send a pull request and contribute to Selenium :)

**Comment by gaoming714** [comment]
@diemol I'd like to create a PR.

**Comment by diemol** [comment]
Great!
If you need a step by step guide, you can check our contribution doc https://www.selenium.dev/documentation/en/contributing/

**Comment by gaoming714** [comment]
Duplicate of #524
I didn't do a PR very skillfully, it is #524.
I hope it works.

**Comment by diemol** [comment]
Closed via #524, thank you!
